config/areas/site/dialogs.php

```diff
@@ -641,13 +641,7 @@
 	'changes' => [
 		'pattern' => 'changes',
 		'load'    => function () {
-			$dialog = new ChangesDialog();
-			return $dialog->load();
+			return (new ChangesDialog())->load();
```

Comment:
`Comment:` Just a future idea to share, not at all related to this PR: I think it would be cool if at one point we have standardized dialog classes and then it would be only 

```php
'changes' => ChangesDialog::class
```

or so - and our Fiber dialog classes would in that case take care of calling the standardized `::pattern()`, `->load()`, `->submit()` etc.

Reply:
I love the idea!